packages/zipkin-encoder-thrift/test/toThrift.test.js

```diff
@@ -0,0 +1,196 @@
+const lolex = require('lolex');
+const toThrift = require('../src');
+const thriftTypes = require('../src/gen-nodejs/zipkinCore_types');
+const {
+  TFramedTransport,
+  TBufferedTransport,
+  TBinaryProtocol
+} = require('thrift');
+const {fromByteArray: base64encode} = require('base64-js');
+const {
+  TraceId,
+  record: {Span, Endpoint},
+  option: {Some, None}
+} = require('zipkin');
+
+describe('Thrift Formatting', () => {
```

Comment:
this was a bear to make, but I noticed some logic only existed in json codec not thrift. I backfilled the tests